content/docs/tutorials/certificate-defaults/README.md

```diff
@@ -0,0 +1,571 @@
+
+```yaml
+cert-manager.io/cluster-issuer: "our-corp-issuer"
+```
+
+This annotation serves as both the trigger for cert-manager to act upon this `Ingress` and also as the configuration value for the `Certificate.spec.issuerRef` fields.
+This single line replaces the need for the user to create a `Certificate` resource entirely. 
+This results in a reduction of the total YAML required to secure the application behind this `Ingress`.
+
+# Summary
+
+This is a fairly simple example of how easy it can be to setup *defaults* for your cluster `Certificate` resources. 
+We've shown how a `ClusterPolicy` doesn't have to "enforce" settings, rather it can be used to set and extend the default options. 
+`Certificate` users can reduce their YAML, whilst maintaining the flexibility to override any value when needed. 
+
+We have shown how a simple `ClusterPolicy` with only 5 rules can change the user experience creating `Certificate` resources from:
+
+```yaml file=../../../../public/docs/tutorials/certificate-defaults/cert-test-revision-override.yaml
+```
+🔗 <a href="cert-test-revision-override.yaml">`cert-test-revision-override.yaml`</a>
+
+To instead only need to specify the configuration important to them, for example:
+
+```yaml file=../../../../public/docs/tutorials/certificate-defaults/cert-test-minimal.yaml
+```
+🔗 <a href="cert-test-minimal.yaml">`cert-test-minimal.yaml`</a>
+
+With this policy we achieved our objective and have enabled users to submit minimal `Certifiate` resources.
+This completes our fifth [use case](#use-cases), with only a single field contained within the specification, the `dnsNames` entry.
+Every other specified field was automatically defaulted using Kyverno with `ClusterPolicy` which would typically be setup by a platform administrator.
+
+# Cleanup
+
+If you created the kind cluster for this tutorial you can simply run:
+
+```shell
+kind delete cluster --name defaults
+```
+
+Otherwise to remove all resources deployed in this tutorial:
+
+```shell
+# Assuming you are running from this directly or saved all the files to yamls/
+kubectl delete -f ingress.yaml
+kubectl delete -f cpol-mutate-certificates-1.yaml
+helm uninstall kyverno -n kyverno-system
+helm uninstall cert-manager -n cert-manager
+helm uninstall ingress-nginx -n ingress-nginx
+```
+
+# Appendix
+
+## cert-manager version requirement
+
+The behavior of cert-manager's mutating webhook has been changed from v1.14.x onward.
+For a more complete explanation and details of the change please refer to [PR #6311](https://github.com/cert-manager/cert-manager/pull/6311).
+Instructions for a manual fix can be found [in this comment on PR #6311](https://github.com/cert-manager/cert-manager/pull/6311#issuecomment-1889517418).
+
+## Presets Feature Request
+
+For further background reading around setting "defaults" or "presets", you can refer to [issue 2239](ttps://github.com/cert-manager/cert-manager/issues/2239).
+This tutorial came out of an investigation of that issue.
```

Comment:
Broken link to CertificatePresets issue

@hawksight I just noticed this broken link while reading the live docs. 
Sorry I missed it during review.